Adyen/Assets/en-US.lproj/Localizable.strings

```diff
@@ -190,13 +190,13 @@
 "adyen.twint.noAppsInstalled.message" = "No or an outdated version of TWINT is installed on this device. Please update or install the TWINT app.";
 "adyen.threeds2.DA.registration.title" = "Secure checkout";
 "adyen.threeds2.DA.biometrics" = "biometric";
-"adyen.threeds2.DA.faceID" = "FaceID";
-"adyen.threeds2.DA.touchID" = "TouchID";
-"adyen.threeds2.DA.opticID" = "OpticID";
-"adyen.threeds2.DA.registration.description" = "You can check out faster next time with this card";
-"adyen.threeds2.DA.registration.firstInfo" = "Enjoy a faster checkout process.";
-"adyen.threeds2.DA.registration.secondInfo" = "Use %@ and Passcode to approve payments.";
-"adyen.threeds2.DA.registration.thirdInfo" = "Remove your credentials at any time.";
+"adyen.threeds2.DA.faceID" = "Face ID";
+"adyen.threeds2.DA.touchID" = "Touch ID";
+"adyen.threeds2.DA.opticID" = "Optic ID";
+"adyen.threeds2.DA.registration.description" = "Check out faster next time with this card";
```

Comment:
@goergisn can I submit these for translation?

Reply:
https://gitlab.is.adyen.com/adyen/streams/checkout/translations/-/merge_requests/42